beta-pipelines/skylab/m3c/CondensedSnm3C.wdl

```diff
@@ -452,8 +454,11 @@ task Split_unmapped_reads {
 
         CODE
 
+        # wait 15 seconds for the files to be written
```

Comment:
was this causing issues? weird

Reply:
yes, the tar command on line 461 kept complaining about the files being changed as it was tarring things up... Im not sure why the tar command would even start up if the previous python command block wasn't finished writing out the files? but the sleep 15 seemed to help